.circleci/tests/test_despike.py

```diff
@@ -17,3 +25,93 @@ def test_data_availability(data_dir, working_dir, output_dir):
     boldfile = data_dir + "/withoutfreesurfer/sub-01/func/" \
         "sub-01_task-mixedgamblestask_run-1_space-MNI152NLin2009cAsym_desc-preproc_bold.nii.gz"
     assert op.exists(boldfile)
+
+
+def test_nifti_despike(data_dir):
+    """Test Nifti despiking."""
```

Comment:
I think it's worth describing the strategy in the docstring as well. It shouldn't go in the summary line, but after that in the detailed summary would be good.

Reply:
Will do!